nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const router = express.Router();
+
+const reservations = require("./../data/reservations.json");
+
+router.get("/", async (request, response) => {
+  try {
+    response.send(reservations)
+    // console.log(reservations);
+    // console.log("in /api/reservations");
+  } catch (error) {
+    throw error;
+  }
+});
+
+router.get("/:id", async(request, response) => {
+  const reservationId = Number(request.params.id);
+  if (!Number.isInteger(reservationId)) {
```

Comment:
This condition checks, whether the given parameter is an integer or not?
It doesn't check whether the given parameter is a valid reservation Id or not!